### ISSUE TITLE
Gnome status API for front end

### ISSUE THREAD

**Issue by tegandbiscuits** [shiftdsm/Galavanting-Gnome#24]
The status should be

* **Dead** if it hasn't gotten a response in X amount of time
* **Abandoned** if it has gotten new locations, but hasn't moved outside of move radius for X amount of time

I'm thinking both times could be the same. Maybe 8, 12, or 24 hours.

**Comment by codebycliff** [comment]
What value do you want returned if either of those states aren't true?

**Comment by tegandbiscuits** [comment]
Thinking about this a bit, it might be good to just modify the `GET /locations` endpoint so it returns something like this.

```
{
  "status": "dead" || "abandoned" || "alive",
  "locations": [
    { "lat": "100.0000", "lon": "100.0000", "published_at": "..." } // + other things
  ]
}
```

It might not be the most semantic, but it would let us figure the status, and where it was.

Arguably this could also just be figured on the front end as well.

**Comment by codebycliff** [comment]
Yeah, I like that better. Probably better off as one endpoint. I think the logic based on location will be advanced enough to keep it on the backend.

**Comment by codebycliff** [comment]
How large do you think the move radius should be?

**Comment by tegandbiscuits** [comment]
Pretty arbitrary but maybe ~2-8km (geolib uses SI)?

I think 2km would be pretty safe, so it'd probably just go with that, at least to get the logic in.
